src/pages/nominate.js

```diff
@@ -52,16 +52,13 @@ class App extends React.Component {
 
   handleSubmit = e => {
     const validation = this.validateName();
+    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to send your nomination')) {
```

Comment:
I think "submit" works better here than "send", but we should probably change the submit button to read something like "Send us a Twitter DM!" to make it clear before they click the button